### DISCUSSION TITLE
REST api v3: Create Pull Request not working anymore?

### DISCUSSION THREAD

**Discussion by exhibiton** [community/community#24308] [Apps, API and Webhooks]
<p>I was working on creating some kind of automation on Github with merging PRs (pretty much just trying to trigger a new Pull Request between staging and testing server each time staging is deployed). </p>
<p>The API in general is working fine with my access key etc, I can view open PRs in the same <code>org_name</code> and <code>repo_name</code> that I am using for POST/Create PR with no problems. But when I try to create a new pull request I keep getting this error: </p>
<p>“message”: “Invalid request. For ‘links/0/schema’, nil is not an object.”</p>
<p>and request I am trying with CURL or Postman is (POST ofc) <a href="https://api.github.com/repos/playven/amperbooking/pulls?access_token=token_that_worked_for_get_private_info&amp;title=test&amp;head=master&amp;base=development" rel="nofollow noopener">https://api.github.com/repos/org_name/repo_name/pulls?access_token=token_that_worked_for_get_private_info&amp;title=test&amp;head=master&amp;base=development</a> and this matches docs for <a href="https://developer.github.com/v3/pulls/#create-a-pull-request" rel="nofollow noopener">https://developer.github.com/v3/pulls/#create-a-pull-request</a> in my opinion. </p>
<p>Has the API changed without the documentation being noticed and this feature is now only supported by the v4 GraphQL API or am I just doing something wrong here?</p>
<p>edit: used hub to do this eventually.</p>

**Comment by jothikanth** [comment, marked as answer]
<p><a class="mention" href="/exhibiton">@exhibiton</a> , I had a similar problem with python requests module and when I used the json keyword over params keyword, it worked. So, the problem is the data you are posting as the status code indicates.</p>

**Comment by Jean-Baptiste-Lasselle** [comment]
<p>hi <a class="mention" href="/jothikanth">@jothikanth</a> , what do you mean by “json keyword over params keyword” ? do you mean the HTTP header set with <code>curl</code> by  <code>-H 'Content-Type: application/json'</code> ?</p>
